src/pages/HomePage.tsx

```diff
@@ -0,0 +1,3 @@
+export const HomePage = () => (
```

Comment:
```suggestion
export const HomePage: FC = () => (
```

Reply:
I am not sure if it is necessary for all pages.